LICENSE

```diff
@@ -54,7 +54,8 @@ Holders for code above are:
   (c) 2010 Xuan Luo
   (c) 2006-2009 Mika Tuupola, Dylan Verheul
   (c) 2010 Giva, Inc.
-  (c) 2012 -2014 Jon Nylander
+  (c) 2012-2014 Jon Nylander
+  (c) 2012-2013 Dave Gandy
```

Comment:
i dont think you need to add this line (Dave Gandy) as its part of font-awesome package afaiu. @dLobatog ?